apps/preview/next/pages/showcases/Card/CardDefault.tsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable jsx-a11y/anchor-has-content */
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { SfButton } from '@storefront-ui/react';
+import earphones from '@assets/card-1.png';
+import eBike from '@assets/card-2.png';
+import boxedWater from '@assets/card-3.png';
+
+const cardDetails = [
+  {
+    image: boxedWater,
+    title: 'Sip Sustainably: The Rise of Boxed Water',
+    description:
+      'Boxed water is a sustainable alternative to traditional plastic bottles, made from renewable resources.',
+    button: 'Read more',
+  },
+  {
+    image: eBike,
+    title: 'Ride the Future: Exploring the Benefits of e-Bikes',
+    description:
+      'Eco-friendly, efficient, and fun modes of transportation that provide a range of benefits for riders and the environment.',
+    button: 'Read more',
+  },
+  {
+    image: earphones,
+    title: 'Unleash the Ultimate Listening Experience',
+    description:
+      'Audiophile headphones offer unmatched sound quality and clarity, making them the go-to choice for music enthusiasts.',
+    button: 'Read more',
+  },
+];
+
+export default function CardDefault() {
+  return (
+    <div className="flex flex-wrap gap-4 lg:gap-6 lg:flex-nowrap">
+      {cardDetails.map(({ image, title, description, button }) => (
+        <div
+          key={title}
+          className="flex flex-col min-w-[325px] max-w-[375px] lg:w-[496px] relative border border-neutral-200 rounded-md hover:shadow-xl"
+        >
+          <a
```

Comment:
Can we use SfLink or not?

Reply:
Good question :) I thought we don't need additional styling that comes with SfLink so decided to use a tag but I can change it :)

Reply:
I think using a separate anchor is good enough. We could use `SfLink`, but there is no need really